src/main/java/com/alkemy/ong/ports/input/rs/api/CategoryApi.java

```diff
@@ -0,0 +1,118 @@
+package com.alkemy.ong.ports.input.rs.api;
+
+import com.alkemy.ong.common.exception.error.ErrorDetails;
+import com.alkemy.ong.ports.input.rs.request.CategoryRequest;
+import com.alkemy.ong.ports.input.rs.response.CategoryNameResponseList;
+import com.alkemy.ong.ports.input.rs.response.CategoryResponse;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.ExampleObject;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.util.Optional;
+
+@Validated
+@SecurityRequirement(name = "bearerAuth")
+public interface CategoryApi {
+
+
+    @Operation(summary = "Get Category by id", responses = {
+            @ApiResponse(responseCode = "200", description = "Successfully retrieved Category", content = {@Content(mediaType = "application/json",
+                    schema = @Schema(implementation = CategoryResponse.class))}),
+            @ApiResponse(responseCode = "400", description = "Invalid request",
+                    content = {@Content(mediaType = "application/json",
+                            schema = @Schema(implementation = ErrorDetails.class))}),
+            @ApiResponse(responseCode = "404", description = "Not found",
+                    content = {@Content(mediaType = "application/json",
+                            examples = @ExampleObject(value = ("Category ID is not found.")),
```

Comment:
Este no es un ejemplo real, sería 

`@ExampleObject(value = "{\"code\":\"RESOURCE_NOT_FOUND\",\"detail\":\"The resource with id 99 is not found\"}"))})`